### PR TITLE
Update: Change styles for marketplace search

### PR DESCRIPTION
**Pull request by patrick-ivers**
#### Changes
- Invert search button colours and make button a square for the marketplace redesign

#### Screenshots
screen shot 2017-05-15 at 2 46 05 pm

### CODE REVIEW

File: src/components/adslotUi/SearchBarComponent.jsx [hunk 1]
```diff
@@ -36,10 +37,12 @@ const SearchBarComponent = ({
         value={searchString}
       />
       <Button
-        className="search-bar-component-button"
+        className="search-bar-component-button btn-inverse"
         bsStyle="primary"
         onClick={onSearch}
-      >&nbsp;</Button>
+      >
+        <SvgSymbol classSuffixes={['search-icon']} href="/assets/svg-symbols.svg#search" />
```
Comment (omgaz) [inline src/components/adslotUi/SearchBarComponent.jsx]: Do we need a test to expect this?

**Comment by patrick-ivers** [comment]
[Marketplace landing: search and filter UI changes.](https://trello.com/c/3QsmbITK/1260-marketplace-landing-search-and-filter-ui-changes)

File: src/components/adslotUi/SearchBarComponent.jsx (hunk 1 above)
Comment (patrick-ivers) [inline src/components/adslotUi/SearchBarComponent.jsx]: Added. 👍

**Comment by sonhanguyen** [comment]
Mergin...

**Comment by patrick-ivers** [comment]
[(3) Marketplace landing: interface - placeholder page](https://trello.com/c/5BgrhBDQ/6232-3-marketplace-landing-interface-placeholder-page)

**Pull request merged**